platform/ios/src/MGLMapView.mm

```diff
@@ -1070,6 +1070,8 @@ - (void)sleepGL:(__unused NSNotification *)notification
     {
         self.dormant = YES;
 
+        [MGLOfflineStorage sharedOfflineStorage].mbglFileSource->setRevalidationState(mbgl::DefaultFileSourceRevalidationState::Inactive);
```

Comment:
Since there may be multiple MGLMapViews but only one MGLOfflineStorage, can we make MGLOfflineStorage observe these notifications to set its own revalidation state?